src/scenes/ApplicationRouter/portal/Review/index.tsx

```diff
@@ -32,6 +80,7 @@ const ReviewView = () => {
 
   return (
     <ApplicationPage crumbs={crumbs}>
+      {selectedApplication.status === 'Submitted' && <ApplicationStatusSubmitted />}
```

Comment:
We can change this to 'In Review' instead. Submitted is an internal one.

The list of statuses are here:
https://terraformation.atlassian.net/wiki/spaces/PM/pages/1295319074/PRD+Application+-+Pre-screen#Data-%2F-Calculations

Reply:
Adding on to this, I think we will need slightly different text/screens for Accepted, Waitlist, Not Accepted as well. But we can follow up with more PRs instead.